app/connectors/CustomsDeclareExportsConnector.scala

```diff
@@ -57,6 +55,11 @@ class CustomsDeclareExportsConnector @Inject()(appConfig: AppConfig, httpClient:
       .map(_.toExportsDeclaration)
   }
 
+  private def logPayload[T](prefix: String, payload: T)(implicit wts: Writes[T]): T = {
+    Logger.debug(s"$prefix: ${Json.toJson(payload)}")
```

Comment:
I'll leave Patryk to comment on the logging (i.e. private val logger = Logger(this.getClass))

Reply:
This was already there. Code reformat made it stand out.
This has been addressed.